dev-cpp/string-theory/string-theory-3.3.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+if [[ ${PV} = 9999 ]]; then
+	  GIT_ECLASS="git-r3"
+	  EXPERIMENTAL="true"
+fi
+
+inherit cmake ${GIT_ECLASS}
+
+DESCRIPTION="A flexible modern C++ library for string manipulation and storage"
+HOMEPAGE="https://github.com/zrax/string_theory/"
+
+SRC_URI="https://github.com/zrax/string_theory/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+S="${WORKDIR}/string_theory-${PV}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Did you test it on both architectures?

Reply:
No - I remove x86 then?

Reply:
Yes please. Or find a x86 to test on, via qemu etc